project/AppDependencies.scala

```diff
@@ -2,21 +2,22 @@ import sbt.*
 
 object AppDependencies {
 
-  private val bootstrapVersion = "7.22.0"
+  private val bootstrapVersion = "8.4.0"
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % bootstrapVersion,
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "7.23.0-play-28",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28" % "1.3.0",
+    play.sbt.PlayImport.ws,
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-29" % bootstrapVersion,
+    "uk.gov.hmrc" %% "play-frontend-hmrc-play-29" % "8.1.0",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-29" % "1.6.0",
```

Comment:
Could you please update to 1.7.0 as it has been updated recently by platops team?